tests/functional/cursor-002.phpt

```diff
@@ -1,236 +0,0 @@
---TEST--
-Sorting single field, ascending, using the Cursor Iterator
---SKIPIF--
-<?php require __DIR__ . "/../utils/basic-skipif.inc"; CLEANUP(STANDALONE) ?>
-<?php $LIMIT = 100; require __DIR__ . "/../utils/fixtures-users.inc" ?>
---FILE--
-<?php
-require_once __DIR__ . "/../utils/basic.inc";
-
-$manager = new MongoDB\Driver\Manager(STANDALONE);
-
-$query = new MongoDB\Driver\Query(array(), array(
-    'projection' => array('_id' => 0, 'username' => 1),
-    'sort' => array('username' => 1),
-));
-
-$qr = $manager->executeQuery(NS, $query);
-$cursor = $qr->getIterator();
-
-var_dump(
-    $cursor->getBatchSize(),
-    $cursor->setBatchSize(15),
-    $cursor->getBatchSize(),
-    $cursor->isDead()
-);
-
-
-
-for($j=0, $cursor->rewind(); $cursor->valid(); $cursor->next(), $j++) {
```

Comment:
This test wasn't relevant anymore, since we're no longer implementing Iterator.

That said, I don't understand why the cursor was dumped on the 10th iteration and we expected `firstBatch` to be:

```
array(1) {
  ["username"]=>
  string(16) "abernathy.audrey"
}
```